extensions/amp-access/0.1/amp-access.js

```diff
@@ -214,6 +217,10 @@ export class AccessService {
       user.warn(TAG, 'Experiment "amp-access-server" is not enabled.');
       type = AccessType.CLIENT;
     }
+    if (type == AccessType.CLIENT && this.isServerEnabled_) {
```

Comment:
This kind of seems big for this PR. I'd not do it now, since it will start encrypting `type=client` reader IDs as well.